helm/nessie/templates/service.yaml

```diff
@@ -39,6 +39,13 @@ spec:
       protocol: TCP
     {{- end }}
   sessionAffinity: {{ .Values.service.sessionAffinity }}
+  {{- if $svc.clusterIP }}
```

Comment:
```suggestion
  {{- if .Values.service }}
```

Reply:
Good catch; I corrected to `.Values.service.clusterIP` though because I don't want the field to appear in the spec at all if it is not specified.